doc/build-netbsd.md

```diff
@@ -1,92 +1,144 @@
-NetBSD Build Guide
-======================
-**Updated for NetBSD [8.0](https://www.netbsd.org/releases/formal-8/NetBSD-8.0.html)**
+# NetBSD Build Guide
 
-This guide describes how to build bitcoind and command-line utilities on NetBSD.
+Updated for NetBSD [9.2](https://netbsd.org/releases/formal-9/NetBSD-9.2.html).
 
-This guide does not contain instructions for building the GUI.
+This guide describes how to build bitcoind, command-line utilities, and GUI on NetBSD.
 
-Preparation
--------------
+## Preparation
 
-You will need the following modules, which can be installed via pkgsrc or pkgin:
+### 1. Install Required Dependencies
+
+Install the required dependencies the usual way you [install software on NetBSD](https://www.netbsd.org/docs/guide/en/chap-boot.html#chap-boot-pkgsrc).
+The example commands below use `pkgin`.
+
+```bash
+pkgin install autoconf automake libtool pkg-config git gmake boost libevent
 
 ```
-autoconf
-automake
-boost
-git
-gmake
-libevent
-libtool
-pkg-config
-python37
 
+See [dependencies.md](dependencies.md) for a complete overview.
+
+### 2. Clone Bitcoin Repo
+
+Now that `git` and the required dependencies are installed, let's clone the Bitcoin Core repository to a directory. All build scripts and commands will run from this directory.
+
+```bash
 git clone https://github.com/bitcoin/bitcoin.git
 ```
 
-See [dependencies.md](dependencies.md) for a complete overview.
+### 3. Install Optional Dependencies
 
-### Building Bitcoin Core
+#### Wallet Dependencies
 
-**Important**: Use `gmake` (the non-GNU `make` will exit with an error).
+It is not necessary to build wallet functionality to run bitcoind or the GUI.
 
-#### With descriptor wallet:
+###### Legacy Wallet Support
+
+`db4` is required to enable support for legacy wallets. Skip if you don't intend to use legacy wallets.
```

Comment:
> Skip if you don't intend to use legacy wallets.

I think sentences like this (repeated for every dependency) are overly verbose (and somewhat obvious) and could be dropped.